blueocean-web/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
so I understand the desire to do it here so stuff is same as remove - ie caching is transparent, but it does mean everyone needs to remember to do it everywhere..